### DISCUSSION TITLE
Black characters with black background I can't read this e-book

### DISCUSSION THREAD

**Discussion by graizzaro** [sumatrapdfreader/sumatrapdf#2845] [Q&A]
I just passed to version 3.4.6 but and I found some ebook I can't read.
![346-1](https://user-images.githubusercontent.com/73635134/182340541-b0d799ac-0231-41a1-bb2d-4317039825b4.jpg)
![346-2](https://user-images.githubusercontent.com/73635134/182340559-56ad8d6b-e993-4cfb-8e8c-447a09429b9f.jpg)

There was no reading problem with 3.3.3 version
![333-1](https://user-images.githubusercontent.com/73635134/182340549-8b4aa3b8-143a-4141-a187-41830892682d.jpg)
![333-2](https://user-images.githubusercontent.com/73635134/182340552-f99f1004-a988-442b-8cd5-ded965ff149a.jpg)

Is there a solution?

Problems with tables in e-book remain.
![333-tab1](https://user-images.githubusercontent.com/73635134/182340562-d08a61a5-5708-4f18-80c2-1a0de2cdfc4c.jpg)
![346-tab1](https://user-images.githubusercontent.com/73635134/182340565-3e62a20b-ee12-4e00-9f0b-3248b72c4510.jpg)
![346-tab2](https://user-images.githubusercontent.com/73635134/182340567-40c3a0fb-d054-4251-a8e5-fa40f2a2f75b.jpg)

Thank-you for your fantastic program

**Comment by GitHubRulesOK** [comment]
always try latest pre-release (portable will do in this case) https://www.sumatrapdfreader.org/prerelease
if not working there then, no sample no fix
If a file is sensitive then email a copy to developer KJK
If file is public then try upload as a zip file into the question

**Reply by graizzaro** [reply]
I've tried with pre-release but I can't solve.

This is the epub file I've tried to read
[1957 Missione su Jaimec (Wasp) URA1532.zip](https://github.com/sumatrapdfreader/sumatrapdf/files/9244461/1957.Missione.su.Jaimec.Wasp.URA1532.zip)

Thank-you for your time

**Reply by GitHubRulesOK** [reply]
the file has invalid colour wh or whi when it should be #ffffff (white)
I simply replaced those invalid entries see if that helps
[1957 Missione su Jaimec (Wasp) URA1532.zip](https://github.com/sumatrapdfreader/sumatrapdf/files/9246087/1957.Missione.su.Jaimec.Wasp.URA1532.zip)

and tables have always been a historic eBook issue

**Reply by graizzaro** [reply]
I've solved.

Thank-you very much
